oeps/oep-0001.rst

```diff
@@ -0,0 +1,334 @@
+the Open edX community first if an idea is original helps prevent too much
+time being spent on something that is guaranteed to be rejected based on
+prior discussions (searching the internet does not always do the trick).
+It also helps to make sure the idea is applicable to the entire community
+and not just the author. Just because an idea sounds good to the author
+does not mean it will work for most Open edX installations.
+
+Once the Author has asked the Open edX community as to whether an idea
+has any chance of acceptance, a draft OEP should be submitted as a pull request
+against the `central OEP repository`_. The pull request title must start
+with "OEP-XXX", and claims that OEP number for the included proposal.
+Before claiming a number, check the list of previous pull requests to
+see which numbers have already been assigned.
+
+.. _central OEP repository: http://github.com/edx/open-edx-proposals
+
+As updates are necessary, the OEP Author can update the pull request.
+
+OEPs can be reference pull requests to other Open edX repositiories which
+will act as reference implementations.
+
+OEP Review & Resolution
+-----------------------
+
+Once the Author has completed an OEP, they may request a review from
+OEP reviewers. The edX Chief Architect will select an Arbiter for the OEP.
+This Arbiter will be recorded in the "Arbiter" header on the OEP. The rest of
+Open edX community will be given the opportunity to comment on the OEP, with the
+Arbiter serving to keep the discussion on track and to evaluate when
+it has reached a final conclusion.
+
+For an OEP to be accepted by the Arbiter, it must meet certain minimum
+criteria. It must be a clear and complete description of the proposed
+enhancement. The enhancement must represent a net improvement. The proposed
+implementation, if applicable, must pass the existing Open edX review process.
+
+An OEP can be assigned the status "Deferred". The OEP Author or Arbiter can
+assign the OEP this status when no progress is being made on the OEP. Once a
+OEP is deferred, the OEP Author can re-assign it to draft status.
+
+An OEP can also be "Rejected" by the Arbiter. Perhaps after all is said and
+done it was not a good idea. It is still important to have a record of this
+fact. The "Withdrawn" status is similar - it means that the OEP Author
+themselves has decided that the OEP is actually a bad idea, or has accepted
+that a competing proposal is a better alternative.
+
+When a OEP is Accepted, Rejected or Withdrawn, the OEP should be updated
+accordingly. In addition to updating the status field, at the very least the
+Resolution header should be added with a link to the relevant post in the edx-code
+mailing list archive or to the appropriate section of the PR.
+
+OEPs can also be superseded by a different OEP, rendering the original
+obsolete.
+
+The possible paths of the status of OEPs are as follows:
+
+.. image:: oep-0001/state-flow.png
```

Comment:
Fundamentally, yes (although the Author can choose to Defer or Withdraw the OEP, for example).